amivapi/auth/link_methods.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+#
+# license: AGPLv3, see LICENSE for details. In addition we strongly encourage
+#          you to buy us beer if we meet and you like the software.
+
+"""Functions adding permitted methods to '_links' part of response.
```

Comment:
the filename is quite generic, it took me some time to figure out what you are dooing here.
Something like authorized_link_methods would probably be more specific.